process/archiving.md

```diff
@@ -0,0 +1,35 @@
+# CNCF Project Archiving Process v1.0
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC.
+
+## Archiving Criteria
+
+There are different criteria to consider when archiving a project, but here are ones that the TOC looks for:
+
+* It hasn't seen a commit in over 6 months.
+* It hasn't seen a release in over 6 months.
+* There haven't been any issues opened for 6 months.
+* Opened issues haven't received a response within 6 months.
+* It's binaries/source are no longer being downloaded
```

Comment:
It might be good to include something in there about its "known usage" by the community. For example, a project could look totally dead (no activity) but it could be very widely used and people have no issues with it. In that case I don't think we'd be wise to archive it. Granted the "download" bullet implies it, but in some cases they might not have downloads, e.g. it's just a spec. Perhaps something like:
> The project does not appear to be widely used by the Cloud Native community.

Reply:
Agreed. If folks stop being dependant on a project and are actively looking to replace, that's a sign of trouble